doc/userguide/src/ExecutingTestCases/TestExecution.rst

```diff
@@ -455,8 +455,88 @@ can be used. The below examples executes all the keywords listed.
        Log   log from keyword 2
 
 
+You can override the recursive continue behavior using the reserved
+`robot:stop-on-failure` keyword tag:
+
+.. sourcecode:: robotframework
+
+   *** Test Cases ***
+   Test
+       [Tags]    robot:recursive-continue-on-failure
+       Should be Equal   1   2
+       User Keyword 1
+       Log   log from test case
+
+   *** Keywords ***
+   User Keyword 1
+       Should be Equal   3   4
+       Log   this is executed
+       User Keyword 2
+       Log   this is also executed
+
+   User Keyword 2
+       [Tags]    robot:stop-on-failure
+       Should be Equal   5   6
+       Log   this is not executed
+
+The `robot:stop-on-failure` and its recursive variant
+`robot:recursive-stop-on-failure` can also be used to alter
+the default continue behavior of Teardowns and Templates:
```

Comment:
Assuming suitable sections exist, it would be good to add links to sections about teardowns and teamplates.